index.md

```diff
@@ -1,19 +1,8 @@
 ---
 layout: HubPage
 hide_bc: true
-
 title: .NET Documentation
-description:
-keywords:
-author:
-manager:
-ms.date: 11/14/2016
-ms.topic: hubpage
-ms.prod:
-ms.service:
-ms.technology:
-ms.suite:
-ms.assetid:
+description: Learn about .NET
```

Comment:
We're trying to include a meta description for SEO purposes, so this is new in the hub page template. If there's nothing better to put there for now, I can remove it.